nixos/modules/services/x11/redshift.nix

```diff
@@ -113,22 +148,42 @@ in {
 
     systemd.user.services.redshift =
     let
-      providerString = if lcfg.provider == "manual"
-        then "${toString lcfg.latitude}:${toString lcfg.longitude}"
-        else lcfg.provider;
+      # This should be appended last in the config file since it introduces
+      # its own section "[manual]"
+      locationString = if lcfg.provider == "manual"
+        then ''
+          [manual]
+          lat=${toString lcfg.latitude}
+          lon=${toString lcfg.longitude}
+        ''
+        else "";
+      fadeString = if cfg.fade then "1" else "0";
+      sectionString = if strings.hasInfix "gammastep" cfg.executable
+        # https://gitlab.com/chinstrap/gammastep/-/commit/1608ed61154cc652b087e85c4ce6125643e76e2f
+        then "[general]"
+        else "[redshift]";
```

Comment:
Not really necessary since `gammastep` still supports `[redshift]`, but this avoids a warning at startup.